packages/abi-gen-wrappers/src/generated-wrappers/asset_proxy_owner.ts

```diff
@@ -699,6 +846,14 @@ export class AssetProxyOwnerContract extends BaseContract {
             const txHash = await self._web3Wrapper.sendTransactionAsync(txDataWithDefaults);
             return txHash;
         },
+        /**
+         * Sends an Ethereum transaction and waits until the transaction has been successfully mined without reverting. If the transaction was mined, but reverted, an error is thrown.
+         * @param assetProxyContract         Address of AssetProxy contract.
+         * @param isRegistered         Status of approval for AssetProxy contract.
+         * @param txData                Additional data for transaction
+         * @param pollingIntervalMs     Interval at which to poll for success
```

Comment:
There are many instances of weird misalignment like this.  I'm wondering whether it's not too unconventional to just put a single space between the parameter name and it's description, and not worry about alignment at all.

Reply:
I don't feel strongly about alignment. Single space looks a bit crowded to me though. Trying 4 spaces